app/templates/apps/new.html

```diff
@@ -28,6 +28,15 @@
       {% endfor %}
     </select>
     <span>/</span>
+
+    <div class="typeahead__container">
+      <div class="typeahead__field">
+        <span class="typeahead__query">
```

Comment:
Is this span `. typeahead__query` necessary, maybe it can be removed and the `<input>` could go directly under `div.typeahead__field` or maybe even have the `__field` class.

Not a problem obviously, I was just curious about this structure :)

Reply:
Very probably, but the default configuration [expects a certain HTML structure](http://www.runningcoder.org/jquerytypeahead/documentation/), and it didn't seem worth the effort to streamline it.

Reply:
Uh, didn't find that in my 2 minutes search earlier (just had a very quick look at the npm module documentation). As I said, not an issue, I was just curious ;)